postcss.config.js

```diff
@@ -0,0 +1,11 @@
+const postcss = require('postcss');
```

Comment:
I think you're right, let's keep this tool simple and get it done. If someone is using PostCSS already then they'll probably be fine without this preprocessor anyway.